section3/exercises/arrays.js

```diff
@@ -107,10 +122,26 @@ var chargerType = "car";
 
 // Write your conditional here
 
+//This if statement is evaluating if the condition percentBatteryLeft is greater
+//than or equal to 15. If it is true, the code will execute the next line.
+if (percentBatteryLeft >= 15)  {
+//If the previous condition was true, this line will execute and will return 'You can call an Uber!'
+  console.log("You can call an Uber!");
+//If initial if statement was evaluated to be false, then the code would continue to
+//the following else statement, which utilizes the comparison operator && to evaluate
+//if both hasCharger returns true and chargerType return car. If that evaluates
+//as true, then the console will log "Heck yeah, you can call an Uber!"
+} else if ((hasCharger === true) && (chargerType === "car")){
```

Comment:
Minor syntax note - we can think of this as 1 conditional, so the inner parenthesis are implicit (they don't change the functionality but they aren't needed)

`(hasCharger === true && chargerType === "car")`